examples/elasticsearch/client.js

```diff
@@ -0,0 +1,53 @@
+import { config } from '@tinyhttp/dotenv'
+import { Client } from '@elastic/elasticsearch'
+config()
+
+// Read from environment
+const ELASTICSEARCH_NODE = process.env.ELASTICSEARCH_NODE
```

Comment:
since those variables are used only in one place you can just do this: `password: process.env....`